client/src/pages/Home.js

```diff
@@ -14,6 +14,7 @@ export function Home() {
 	const [isFailure, setIsFailure] = useState(false);
 	const [isUploadSuccess, setIsUploadSuccess] = useState(false);
 	const [isUploadFailure, setIsUploadFailure] = useState(false);
+	const [isHiden, setIsHiden]= useState(null);
```

Comment:
```suggestion
	const [expandedRowIndex, setExpandedRowIndex]= useState(null);
```
A variable that starts with `is` suggests that it holds a boolean value, but `isHiden` doesn't. It also doesn't hold a value that represents a hidden row—instead it holds a value that represents the expanded row.

This state contains the index of the row that's expanded, so what do you think about renaming it to something like `expandedRowIndex`?

Also, react state should usually live closest to the component that uses it. `Home` doesn't use the react state—we just pass it straight to `MainTable`. Since it's used only in `MainTable`, let's move this state to `MainTable`

Reply:
Hey Coung I do not have this kind thougts when given a name but I will take it in consideration for future names and use cases.